lms/djangoapps/courseware/views/views.py

```diff
@@ -956,6 +956,25 @@ def course_about(request, course_id):
         # get prerequisite courses display names
         pre_requisite_courses = get_prerequisite_courses_display(course)
 
+        courses_requirements_not_met = get_pre_requisite_courses_not_completed(request.user, frozenset({course.id}))
+
+        run_extension_point(
```

Comment:
Why don't you explicitly update `courses_requirements_not_met`?. @anfbermudezme 
```suggestion
        courses_requirements_not_met = run_extension_point(
```